packages/p2p/src/components/modal-manager/modals/ad-visibility-error-modal/ad-visibility-error-modal.tsx

```diff
@@ -1,54 +1,47 @@
 import React from 'react';
 import { Button, Modal, Text } from '@deriv/components';
-import { observer } from 'mobx-react-lite';
+import { observer } from '@deriv/stores';
 import { localize, Localize } from 'Components/i18next';
-import { useStores } from 'Stores';
 import { useModalManagerContext } from 'Components/modal-manager/modal-manager-context';
-import { api_error_codes } from '../../../../constants/api-error-codes';
+import { api_error_codes } from 'Constants/api-error-codes';
+import { useStores } from 'Stores';
 
-const AdVisibilityErrorModal = ({ error_code }) => {
-    const { my_ads_store } = useStores();
+type TAdVisibilityErrorModalProps = {
+    error_code: string;
```

Comment:
@nada-deriv like this?
```suggestion
    error_code: api_error_codes.AD_EXCEEDS_BALANCE | api_error_codes.AD_EXCEEDS_DAILY_LIMIT;
```

Reply:
hm.. i think we can leave it as a string itself.. may be new error codes might come up in the future.. and we can ask for the general message design later on in the future if by any chance we get the default message..

Reply:
Sure 👍